src/es5processor.ts

```diff
@@ -284,6 +289,11 @@ class ES5Processor extends Rewriter {
     return this.isRequire(arg as ts.CallExpression);
   }
 
+  isEsModuleProperty(expr: ts.ExpressionStatement): boolean {
+    // We're matching the explicit source text generated by the TS compiler.
+    return expr.getText() === 'Object.defineProperty(exports, "__esModule", { value: true });';
```

Comment:
did you consider matching the tree a bit more deeply? What if they change the whitespace around the brackets later? I'm on the fence about whether it would be better to check eg.
```
expr.LHS = 'Object'
expr.RHS = 'defineProperty'
expr.RHS.arguments = ['exports', '__esModule', *anything*]
```